app/assets/stylesheets/application.scss

```diff
@@ -37,14 +37,28 @@ strong {
   font-weight: 700;
 }
 
+// .nav-wrapper {
+//   opacity: 0.2;
+// }
+
 nav {
-  background-color: rgb(100, 100, 100);
-  opacity: 0.3;
+  background-color: rgb(255, 255, 255);
```

Comment:
Color literals like `rgb(255, 255, 255)` should only be used in variable declarations; they should be referred to via variable everywhere else.